packages/core-entities/src/models/views/YieldViews.ts

```diff
@@ -603,6 +637,29 @@ export const YieldViews = (self: Instance<typeof NetworkModel>) => {
     }
   };
 
+  const getPortfolioStateZeroEarnData = () => {
+    const productGroupData = [getFCashLendData(), getPrimeCashLendData(), getNTokenData()];
+    const tokenList = getUniqueUnderlyingSymbols(productGroupData);
+    const highestAPYSymbol = getDefaultHighestAPYSymbol(productGroupData);
+
+    return {
+      tokenList,
+      productGroupData,
+      defaultSymbol: highestAPYSymbol,
+    };
+  };
+
+  const getPortfolioStateZeroBorrowData = () => {
+    const productGroupData = [getPrimeCashDebtData(), getFCashDebtData()];
```

Comment:
Instead of making this an array of arrays, why not use a key value structure?

Reply:
These are are moved to the portfolio store in the next PR. They are in a array because there are times that we need to loop over all of them and add them up or do different data manipulation and the array made it easier